node/src/test_utils/persistent_configuration_mock.rs

```diff
@@ -639,3 +627,13 @@ impl PersistentConfigurationMock {
         }
     }
 }
+
+pub fn encrypted_past_neighbors(
+    cryptde: &dyn CryptDE,
+    past_neighbors: &str,
+) -> Vec<NodeDescriptor> {
+    past_neighbors
+        .split(",")
```

Comment:
We probably accept also syntax with more than one spaces. Although, I don't think somebody would be so absent-minded to do this in a test and if so he would probably soon understand what advert consequences it brought to him.

Reply:
I don't feel like tracking it down, but I _hope_ we don't accept spaces in there.

Reply:
Yeah. You did right. I talked silly. First of all, it doesn't seem realistic the user has any official channel to set this parameter in the database... and so I probably was thinking of something else while now it seems clear to me. Never mind.